doc/webservices/extract_api.rst

```diff
@@ -0,0 +1,478 @@
+0              Success
+1              Not ready
+2              An error occured
+9              Server is currently under maintenance. Please try again later.
+=============  ==============================================================
+
+.. rst-class:: setup doc-aside
+
+.. switcher::
+
+    .. code-block:: json
+
+        {
+            "jsonrpc": string,
+            "id": string,
+            "result": {
+                "document_id_1": {
+                    "status_code": int,
+                    "status_msg": str,
+                    "results": [{"feature_1_name": feature_1_result, 
+                                 "feature_2_name": feature_2_result,
+                                 …
+                                }]
+                    },
+                "document_id_2": {
+                    "status_code": int,
+                    "status_msg": str,
+                    "results": [{"feature_1_name": feature_1_result, 
+                                 "feature_2_name": feature_2_result,
+                                 …
+                                }]
+                    },
+                    ...
+            }
+        }
+
+.. _webservices/extract_api/invoice_get_results/feature_result:
+
+``feature_result``:
+'''''''''''''''''''
+
+* ``selected_value``: the best candidate for this feature.
+* ``words``: list of all the candidates for this feature ordered by decreasing score.
+
+.. container:: doc-aside
+
+    .. switcher::
+
+        .. code-block:: json
+
+            {
+                "selected_value": candidate_12,
+                "words": [candidate_12, candidate_3, candidate_4,...]
+            }
+
+
+``candidate``:
+''''''''''''''
+
+* ``content``: representation of the candidate.
+* ``coords``: ``[center_x, center_y, width, height, rotation_angle]``. The position and dimensions are relative to the size of the page and are therefore between 0 and 1. The angle is a clockwise rotation measured in degrees.  
+* ``page``: page on which the candidate is located (starts at 0).
+
+.. container:: doc-aside
+
+    .. switcher::
+
+        .. code-block:: json
+
+            {
+                "content": string|float,
+                "coords": [float, float, float, float, float],
+                "page": int
+            }
+
++---------------------+------------------------------------------------------------------------------------+
+| Feature name        | Specifities                                                                        |
++=====================+====================================================================================+
+| SWIFT_code          | **content** is a dictionary encoded as a string.                                   |
+|                     |                                                                                    |
+|                     | It contains information about the detected SWIFT code (also called BIC).           |
```

Comment:
```suggestion
|                     | It contains information about the detected SWIFT code (or BIC).           |
```
Might be worth linking to the ISO 9362 wiki page for clarification / explanation?

Reply:
Ok